src/napari/_vispy/overlays/base.py

```diff
@@ -62,67 +60,17 @@ class VispyCanvasOverlay(VispyBaseOverlay):
     def __init__(self, *, overlay, node, parent=None) -> None:
         super().__init__(overlay=overlay, node=node, parent=parent)
 
-        # offsets and size are used to control fine positioning, and will depend
+        # size is used to control fine positioning, and will depend
```

Comment:
I think this would be important to call out in the class docstring too. It's a bit ambiguous what subclass refers to: a subclass of `VispyCanvasOverlay` or `VispyBaseOverlay`?